rep-2000.rst

```diff
@@ -1188,114 +1188,11 @@ Build System Support:
 Kilted Kaiju (May 2025 - November 2026)
 ---------------------------------------
 
-Targeted platforms:
-
-+--------------+------------------+---------------+------------------+-------------+-----------------+----------------+
-| Architecture | Ubuntu Noble     | Windows 10    | RHEL 9           |  macOS      | Debian Bookworm | OpenEmbedded / |
-|              | (24.04)          | (VS2019)      |                  |             | (12)            | Yocto Project  |
-+==============+==================+===============+==================+=============+=================+================+
-|    amd64     | Tier 1 [d][a][s] | Tier 1 [a][s] | Tier 2 [d][a][s] |  Tier 3 [s] |   Tier 3 [s]    |   Tier 3 [s]   |
-+--------------+------------------+---------------+------------------+-------------+-----------------+----------------+
-|    arm64     | Tier 1 [d][a][s] |               |                  |             |   Tier 3 [s]    |   Tier 3 [s]   |
-+--------------+------------------+---------------+------------------+-------------+-----------------+----------------+
-|    arm32     |    Tier 3 [s]    |               |                  |             |   Tier 3 [s]    |   Tier 3 [s]   |
```

Comment:
The Ubuntu Noble (24.04) arm32 Tier 3 [s] target platform isn't noted in the new table. Is this intentional? I know you mentioned you intentionally left things out during the PMC meeting yesterday; not sure if this was one of them.